src/stories/theme.js

```diff
@@ -17,5 +17,8 @@ const theme = {
   badgeBg: "#FF3D00",
   badgeText: "#fff",
   white: "#f6f9fc",
+  background: {
+    outlined: "transparent",
+  },
```

Comment:
do you need this?

Reply:
no sir i have change my code in style component only i will update this part